dockbarx/groupbutton.py

```diff
@@ -2203,6 +2202,8 @@ def show(self, delay=None, force=False):
         except:
             logger.exception("If an empty popup was shown this " + \
                              "might have somethin to do with it:")
+        for window in group:
+            window.item.set_show_preview(True)
```

Comment:
What does these lines do? They makes dockbarx ignore the option "Show previews" in preferences dialog and always shows the previews no matter what, but do they serve any other purpose?

Reply:
Removing them seems to make the option "Show previews" work again but I don't want to remove the lines if the serve another purpose as well.

Reply:
Nice catch, thanks! I'll fix it in next pull request.